sale_coupon_advanced/tests/test_sale_coupon_forced_reward_product.py

```diff
@@ -96,6 +97,8 @@ def test_02_order_line_forced_reward_result(self):
         # Case 1.
         order = self.order_forced
         order.recompute_coupon_lines()
```

Comment:
```suggestion
```

Reply:
@i-vyshnevska I recompute twice on purpose. There was a bug, that would duplicate lines if you recompute multiple times. So this is done to make sure it is fixed.